dcspy/aircraft.py

```diff
@@ -78,13 +79,26 @@ def _load_plane_yaml(self) -> None:
                     request = request.replace('|', '\n|')
                     request = request.strip('|')
                     button_actions[key] = request
+                elif 'BUTTON' in request:
+                    request = request.split(' BUTTON')[0]
+                    push_buttons[key] = request
```

Comment:
We only care about the control as the `0` or `1` value will be provided by the callback handler for the key up/down event.

Reply:
I think we can remove `push_buttons` field from Aircraft and handle like `button_actions`